sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -55,6 +87,19 @@ class EventHubProducerClient(ClientBase):
     :keyword str connection_verify: Path to the custom CA_BUNDLE file of the SSL certificate which is used to
      authenticate the identity of the connection endpoint.
      Default is None in which case `certifi.where()` will be used.
+    :keyword bool enable_idempotent_partitions: Indicates whether or not the producer should enable idempotent
+     publishing to the Event Hub partitions. If enabled, the producer will only be able to publish directly
+     to partitions; it will not be able to publish to the Event Hubs gateway for automatic partition routing
+     nor using a partition key. Default is False.
+    :keyword partition_configs: The set of configurations that can be specified to influence publishing behavior
```

Comment:
If we use "config" - I would make it singular.

Reply:
it's actually composed of multiple partition configs -- a dictionary, the concern I have is that singular is not precise.

maybe it should be called `partitions_config`?

Reply:
name updated